featurebyte/query_graph/transform/decompose_point.py

```diff
@@ -182,16 +182,10 @@ def extract_from_target_node(self, node: Node) -> Optional[FeatureJobSetting]:
         -------
         Optional[FeatureJobSetting]
         """
-        feature_job_settings = set()
         for _node in self.graph.iterate_nodes(target_node=node, node_type=None):
             if isinstance(_node, AggregationOpStructMixin):
-                fjs = self.extract_from_agg_node(node=_node)
-                if fjs:
-                    feature_job_settings.add(fjs)
-
-        if len(feature_job_settings) > 1:
-            assert False, "Multiple feature job settings found"
-        return next(iter(feature_job_settings), None)
+                return self.extract_from_agg_node(node=_node)
+        return None
```

Comment:
Update the logic here so that only the first feature job setting from the aggregation node is considered.